common/app/common/configuration.scala

```diff
@@ -368,6 +368,11 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
     lazy val key = playConfiguration.getStringFromStage("media.key");
   }
 
+  object switchBoard {
+    val bucket = playConfiguration.getMandatoryStringFromStage("aws.switchesBucket")
```

Comment:
let's call them

```
switchboard.bucket  (instead of aws.switchesBucket)
switchboard.object   (instead of newSwitches.key)
```

`key` gives me more the idea of an api key, not an object key in S3